perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_facility_attribute_def_def_pbsServer.java

```diff
@@ -37,7 +33,7 @@ public void checkAttributeSemantics(PerunSessionImpl perunSession, Facility faci
 				break;
 			}
 		}
-		if(!success) throw new WrongAttributeValueException(attribute, "There is no such facility with the same name like this pbsServer");
+		if(!success) throw new WrongReferenceAttributeValueException(attribute, null, facility, null, "No core attribute " + pbsServer + " exist for facility object in perun.");
```

Comment:
This exception is still not what I meant. You should say that there is no facility with the "urn:..facility:..def:core:name" same as the pbsServer attribute. That's what I meant by core attribute.